generators/client/templates/react/src/main/webapp/app/shared/util/date-utils.ts.ejs

```diff
@@ -25,3 +25,5 @@ export const convertDateTimeFromServer = date =>
 
 export const convertDateTimeToServer = date =>
   (date ? moment(date, APP_LOCAL_DATETIME_FORMAT_Z).toDate() : null);
+
+export const defaultDateTime = () => moment().startOf('day').format(APP_LOCAL_DATETIME_FORMAT);
```

Comment:
it's a function, please use a verb to mark the action (like convertXXX, whatever).

Reply:
ok